MicrosoftAlwaysOnVPN: add documentation (#549)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/network/microsoft_always_on_vpn.md

uuid: 80b8382e-0667-4469-bbc9-74be1e0ca1c1
name: Microsoft Always On VPN
type: intake

## Overview
Microsoft Always On VPN is a Windows 10 feature enabling secure connection to protected networks.

This guide will explain how to forward Network Policy Server (NPS) logs to SEKOIA.IO

{!_shared_content/operations_center/detection/generated/suggested_rules_80b8382e-0667-4469-bbc9-74be1e0ca1c1_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/microsoft-always-on-vpn_do_not_edit_manually.md!}

## Configure

### NXLog agent to the Rsyslog

#### Configuring NPS logging

Refer to the [Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/networking/technologies/nps/nps-accounting-configure#configure-nps-log-file-properties) to activate the NPS logging on the server. The logs should be write in files with the IAS (legacy) format.

#### NXLog setup on Windows

> This section describes how to configure [NXLog](https://nxlog.co/) to forward your Windows events by means of a syslog transport channel.

Reviewer: ```suggestion
This section describes how to configure [NXLog](https://nxlog.co/) to forward your Windows events through syslog.
```

To simplify
